diff --git a/node_modules/isomorphic-ws/browser.js b/node_modules/isomorphic-ws/browser.js
index f59be2d..4d0b4df 100644
--- a/node_modules/isomorphic-ws/browser.js
+++ b/node_modules/isomorphic-ws/browser.js
@@ -1,17 +1,5 @@
 // https://github.com/maxogden/websocket-stream/blob/48dc3ddf943e5ada668c31ccd94e9186f02fafbd/ws-fallback.js
 
-var ws = null
-
-if (typeof WebSocket !== 'undefined') {
-  ws = WebSocket
-} else if (typeof MozWebSocket !== 'undefined') {
-  ws = MozWebSocket
-} else if (typeof global !== 'undefined') {
-  ws = global.WebSocket || global.MozWebSocket
-} else if (typeof window !== 'undefined') {
-  ws = window.WebSocket || window.MozWebSocket
-} else if (typeof self !== 'undefined') {
-  ws = self.WebSocket || self.MozWebSocket
-}
-
-export default ws
+const globalObject = typeof window !== 'undefined' ? window : (typeof self !== 'undefined' ? self : {});
+export const WebSocket = globalObject.WebSocket || globalObject.MozWebSocket;
+export default WebSocket;
